src/data/CONTRIBUTORS.ts

```diff
@@ -38,6 +38,13 @@ const CONTRIBUTORS = ensureRecord<Contributor>()({
 			JOBS.DRAGOON,
 		],
 	},
+	KELOS: {
+		name: 'Kelos',
+		avatar: '',
```

Comment:
Just double-checking that you meant not to include an avatar icon here. Changes otherwise look fine, so if you're ok without having the icon, we can merge this in.

Reply:
Yes, skipping the icon was intentional.